.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,9 +34,9 @@ Here are some guidelines to help the review process go smoothly.
    features or make changes out of the scope of those requested by the reviewer
    (doing this just add delays as already reviewed code ends up having to be
    re-reviewed/it is hard to tell what is new etc!). Further, please do not
-   rebase your branch on master/force push/rewrite history, doing any of these
+   rebase your branch on main/force push/rewrite history, doing any of these
```

Comment:
We really use branch names like `branch-24.08`. If we want to change this, we should just remove the specific branch names.
```suggestion
   rebase your branch/force push/rewrite history, doing any of these
```